src/exchange/exchange.go

```diff
@@ -64,11 +65,12 @@ type Exchange struct {
 // Config exchange config struct
 type Config struct {
 	Rate                    string // SKY/BTC rate, decimal string
+	EthRate                 string // SKY/ETH rate, decimal string
 	TxConfirmationCheckWait time.Duration
 }
 
 // NewExchange creates exchange service
-func NewExchange(log logrus.FieldLogger, store Storer, scanner scanner.Scanner, sender sender.Sender, cfg Config) (*Exchange, error) {
+func NewExchange(log logrus.FieldLogger, store Storer, scanner, ethScanner scanner.Scanner, sender sender.Sender, cfg Config) (*Exchange, error) {
```

Comment:
We need a multiplexer for scanners.

Another object that implements scanner.  It calls GetDeposit from all of the other scanners, andputs the deposits into one channel. Then exchange reads from the multiplexer's deposit channel.